src/login/refresh-token.jsx

```diff
@@ -0,0 +1,11 @@
+const refreshToken = async(handle, refreshTiming) => {
+  const newAuthRes = await handle.reloadAuthResponse()
+  refreshTiming = (newAuthRes.expires_in || 3600 - 5 * 60) * 1000
+  setTimeout(refreshToken, refreshTiming)
+}
+
+export const Refreshtoken = handle => {
```

Comment:
There's no reason for this function to be uppercased, as it's not a component. :smile:

Reply:
Done